src/pylorax/api/dnfbase.py

```diff
@@ -56,7 +56,8 @@ def get_base_object(conf):
     dbc.reposdir = [repodir]
     dbc.install_weak_deps = False
     dbc.prepend_installroot('persistdir')
-    dbc.tsflags.append('nodocs')
+    # tsflags is append option therefor the action result in append operation into tsflags
+    dbc.tsflags = ('nodocs', )
```

Comment:
This should be dbc.tsflags += ('nodocs',) since we don't want to lose whatever else is set there.